constants/deployments-mainnet-cron.json

```diff
@@ -0,0 +1,53 @@
+{
+  "kovan": {
+    "vaults": {
+      "RibbonThetaVaultETHCall": {
+        "address": "0xF77FE1e54bc308C88CBacf3670EB54ce4A41226E",
+        "strikeSelection": "0xaAA24915AfFab8A83af74D5A1472F4fE7AD08D71",
+        "optionsPremiumPricer": "0xE922806E610A6bA9A828a3b2Ea6A9936182FFC7D"
+      },
+      "RibbonThetaVaultWBTCCall": {
+        "address": "0x0E24525cCb27d20354a34652f43cF707D8F0Cc97",
+        "strikeSelection": "",
+        "optionsPremiumPricer": "0x947398a2C3f5dD6983bb935FEBFcF076C870e23e"
+      },
+      "RibbonThetaVaultETHPut": {
+        "address": "0xEC1c50724CF7a618C6cdA6CFEa5c9064AFc98E84",
+        "strikeSelection": "0xaAA24915AfFab8A83af74D5A1472F4fE7AD08D71",
+        "optionsPremiumPricer": "0xE922806E610A6bA9A828a3b2Ea6A9936182FFC7D"
+      }
+    },
+    "gnosisLibrary": "",
+    "vaultLifecycle": "0x9bCF7E35E8B852de70ddB993BC461d1f4624e5AC",
+    "univ3pools": {
+      "wethusdc": "",
+      "wbtcusdc": ""
+    }
+  },
+
```

Comment:
prefer this custom heirarchical json to read from in master cron. will need to update addresses manually when we deploy new vaults